File: src/kyu_6/string_array_duplicates/cpp/string_array_duplicates_test.cpp
#include<vector>
using namespace std;

Describe(String_array_revisal)
{
    It(Example_tests)
    {
        Assert::That(dup(vector<string>{"ccooddddddewwwaaaaarrrrsssss","piccaninny","hubbubbubboo"}), Equals(vector<string>{"codewars","picaniny","hubububo"}));
        Assert::That(dup(vector<string>{"abracadabra","allottee","assessee"}), Equals(vector<string>{"abracadabra","alote","asese"}));
        Assert::That(dup(vector<string>{"kelless","keenness"}), Equals(vector<string>{"keles","kenes"}));
   
    }
}; 

File: src/kyu_6/string_array_duplicates/csharp/string_array_duplicates_test.cs
using NUnit.Framework;
using System;
 
[TestFixture]
public class SolutionTest
{
  [Test]
  public void ExampleTests()
  {
    Assert.AreEqual(new String[]{"codewars","picaniny","hubububo"}, Solution.dup(new String[]{"ccooddddddewwwaaaaarrrrsssss","piccaninny","hubbubbubboo"}));
    Assert.AreEqual(new String[]{"abracadabra","alote","asese"}, Solution.dup(new String[]{"abracadabra","allottee","assessee"}));       
    Assert.AreEqual(new String[]{"keles","kenes"}, Solution.dup(new String[]{"kelless","keenness"}));
   
  }
}